src/admin.c

```diff
@@ -1331,21 +1332,21 @@ static bool admin_cmd_enable(PgSocket *admin, const char *arg)
 }
 
 
-static PgSocket *find_socket_in_list(PgSocket *target_client, struct StatList *sockets)
+static PgSocket *find_socket_in_list(unsigned long long int target_client_id, struct StatList *sockets)
 {
 	struct List *item;
 	PgSocket *socket;
 
 	statlist_for_each(item, sockets) {
 		socket = container_of(item, PgSocket, head);
-		if (target_client == socket) {
+		if (target_client_id == socket->id) {
 			return socket;
 		}
 	}
 	return NULL;
 }
 
-static PgSocket *find_client_global(PgSocket *target_client)
+static PgSocket *find_client_global(unsigned long long int target_client_id)
```

Comment:
Ditto. `target_id`.

Reply:
Changed all instances in src/admin.c of target_client_id to target_id.